src/language-js/print/statement.js

```diff
@@ -168,14 +168,19 @@ const isClassProperty = ({ type }) =>
   type === "ClassProperty" ||
   type === "PropertyDefinition" ||
   type === "ClassPrivateProperty" ||
-  type === "ClassAccessorProperty";
+  type === "ClassAccessorProperty" ||
+  type === "AccessorProperty" ||
+  type === "TSAbstractAccessorProperty";
 /**
  * @returns {boolean}
  */
 function shouldPrintSemicolonAfterClassProperty(node, nextNode) {
   const name = node.key && node.key.name;
```

Comment:
Since we are here. Let's remove this chaining, I don't think we key can be nullish here.

```suggestion
  const { name } = node.key;
```